.github/workflows/pr-build.yml

```diff
@@ -68,8 +68,23 @@ jobs:
         key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
         restore-keys: |
           ${{ runner.os }}-node-
-    - name: test
+    - name: Test CDK
       working-directory: cdk_infra/
       run: |
         npm install
         npm run test
+  adot-testbed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: adot-testbed build
```

Comment:
you need to setup the credentials otherwise the testbed will not be able to access cw logs.

Reply:
resolving as we will not run the test